Tests/winswTests/Util/CLITestHelper.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.IO;
+using JetBrains.Annotations;
+using winsw;
+
+namespace winswTests.Util
+{
+    /// <summary>
+    /// Helper for WinSW CLI testing
+    /// </summary>
+    public static class CLITestHelper
```

Comment:
agreed.  Once the restructuring is complete - we can then also do the restructure of the test code.